bin/p2-inspect/main.go

```diff
@@ -58,7 +58,7 @@ func main() {
 	if err != nil {
 		message := "Could not list reality kvpairs: %s"
 		if kvErr, ok := err.(consulutil.KVError); ok {
```

Comment:
is this if statement necessary? what's the difference between putting a `consulutil.KVError` through `%s`